lib/active_shipping/shipping/carriers/ups.rb

```diff
@@ -45,15 +45,15 @@ class UPS < Carrier
           :occasional
         end
       end
-      
+
       DEFAULT_SERVICES = {
         "01" => "UPS Next Day Air",
-        "02" => "UPS 2nd Day Air",
+        "02" => "UPS Second Day Air",
         "03" => "UPS Ground",
         "07" => "UPS Worldwide Express",
         "08" => "UPS Worldwide Expedited",
         "11" => "UPS Standard",
-        "12" => "UPS 3 Day Select",
+        "12" => "UPS Three-Day Select",
```

Comment:
The same here: https://www.ups.com/content/us/en/shipping/time/service/three_day.html?WT.mc_id=VAN701413